### ISSUE TITLE
failed compile on 'Building Python wrapper library for gazeinfer'

### ISSUE THREAD

**Issue by ynott** [Ryoyo-NV/Gaze-Analysis-System#3]
# Environment:
  - HW: Jetson Xavier NX
  - OS: JetPack 4.6.1  

# Expected behaviour

The following Install procedure completes without problems.
https://github.com/Ryoyo-NV/Gaze-Analysis-System#installation

# Actual behaviour

The compile of gazeinfer will fail

```
Building Python wrapper library for gazeinfer...
Cloning into 'deepstream_tao_apps'...
remote: Enumerating objects: 1248, done.
remote: Counting objects: 100% (187/187), done.
remote: Compressing objects: 100% (135/135), done.
remote: Total 1248 (delta 89), reused 114 (delta 52), pack-reused 1061
Receiving objects: 100% (1248/1248), 26.31 MiB | 7.28 MiB/s, done.
Resolving deltas: 100% (646/646), done.
g++ -c -o deepstream_faciallandmark_meta.o -Wall -Werror -DPLATFORM_TEGRA -I /opt/nvidia/deepstream/deepstream/sources/includes -I /opt/nvidia/deepstream/deepstream/sources/includes/cvcore_headers -I /usr/local/cuda-10.2/include -I deepstream_tao_apps/apps/tao_others/deepstream-faciallandmark-app -I deepstream_tao_apps/apps/tao_others/deepstream-gaze-app `pkg-config --cflags gstreamer-1.0 python3 pygobject-3.0` -D_GLIBCXX_USE_CXX11_ABI=1 -Wno-sign-compare -fPIC deepstream_tao_apps/apps/tao_others/deepstream-faciallandmark-app/deepstream_faciallandmark_meta.cpp
g++ -c -o call_probes_from_py.o -fpermissive -Wall -Werror -DPLATFORM_TEGRA -I /opt/nvidia/deepstream/deepstream/sources/includes -I /opt/nvidia/deepstream/deepstream/sources/includes/cvcore_headers -I /usr/local/cuda-10.2/include -Ideepstream_tao_apps/apps/tao_others/deepstream-faciallandmark-app -I deepstream_tao_apps/apps/tao_others/deepstream-gaze-app `pkg-config --cflags gstreamer-1.0 python3 pygobject-3.0` -D_GLIBCXX_USE_CXX11_ABI=1 -Wno-sign-compare -fPICcall_probes_from_py.cpp
cc -shared -o dscprobes.so deepstream_faciallandmark_meta.o call_probes_from_py.o `pkg-config --libs gstreamer-1.0 python3 pygobject-3.0` -L/opt/nvidia/deepstream/deepstream/lib/ -lnvdsgst_meta -lnvds_meta -lnvds_inferutils -lnvds_utils -lm -lstdc++ -L/usr/local/cuda-10.2/lib64/ -lcudart -lcuda -L/opt/nvidia/deepstream/deepstream/lib/cvcore_libs -lnvcv_faciallandmarks -lnvcv_core -lnvcv_tensorops -lnvcv_trtbackend -Wl,-rpath,/opt/nvidia/deepstream/deepstream/lib/
done.

Building gazeinfer library...
-fPIC -DDS_VERSION="6.1.0" -I /usr/local/cuda-10.2/include -I ../../../includes -I /opt/nvidia/deepstream/deepstream/sources/gst-plugins/gst-nvdsvideotemplate/includes -I /opt/nvidia/deepstream/deepstream/sources/includes -I /opt/nvidia/deepstream/deepstream/sources/includes/cvcore_headers -I ../../deepstream-faciallandmark-app -I ../ -I ../../common/ -pthread -I/usr/include/gstreamer-1.0 -I/usr/include/orc-0.4 -I/usr/include/gstreamer-1.0 -I/usr/include/glib-2.0-I/usr/lib/aarch64-linux-gnu/glib-2.0/include -Wno-deprecated-declarations
g++ -c -o gazeinfer.o -fPIC -DDS_VERSION=\"6.1.0\" -I /usr/local/cuda-10.2/include -I ../../../includes -I /opt/nvidia/deepstream/deepstream/sources/gst-plugins/gst-nvdsvideotemplate/includes -I /opt/nvidia/deepstream/deepstream/sources/includes -I /opt/nvidia/deepstream/deepstream/sources/includes/cvcore_headers -I ../../deepstream-faciallandmark-app -I ../ -I ../../common/ -pthread -I/usr/include/gstreamer-1.0 -I/usr/include/orc-0.4 -I/usr/include/gstreamer-1.0-I/usr/include/glib-2.0 -I/usr/lib/aarch64-linux-gnu/glib-2.0/include -Wno-deprecated-declarations gazeinfer.cpp
In file included from gazeinfer.cpp:52:0:
../../common/ds_yml_parse.h:32:10: fatal error: gst/rtsp-server/rtsp-server.h: No such file or directory
 #include <gst/rtsp-server/rtsp-server.h>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
Makefile:70: recipe for target 'gazeinfer.o' failed
make: *** [gazeinfer.o] Error 1
Setup failed. Please check error messages.
exit.
```

# Step reprocedure.

1. Run upgrade
  `sudo apt upgrade`
3. Install Deepstream-6.0.1
  `sudo apt install -y deepstream-6.0`
4. Clone Git Repository & exec setup.sh
```shell
git clone https://github.com/Ryoyo-NV/Gaze-Analysis-System
chmod +x setup.sh
./setup.sh
```

**Comment by ynott** [comment]
This error was corrected by installing the following package.

`libgstrtspserver-1.0-dev`

```
$ sudo apt install -y libgstrtspserver-1.0-dev
```

**Comment by takutoi** [comment]
Thank you for your report.
It's probably due to some changes for deepstream_tao_apps with release of DS6.1 a few days ago.
I will fix this in the near.

**Comment by takutoi** [comment]
fixed by #5
